crm_rma_prodlot_invoice/README.md

```diff
@@ -0,0 +1,18 @@
+CRM Claim Prodlot Invoice
+=========================
+
+When invoice is created from sale order, the move_id field
+filled with information in picking and procurement.
+
+It mainly contains the following features:
+------------------------------------------
+
+* Filled move_id in lines of invoice when sale order is validated and invoice is created before picking
+* Filled move_id in lines of invoice when sale order is validated and picking is created before invoice
```

Comment:
It's not duplicated. 

"Filled move_id in lines of invoice when sale order is validated and invoice is created before picking" ( when in a sale order the invoice is created before of to create the picking. When it's create the picking, action_ship_create method is called and the move_id created is related with the invoice )

+* Filled move_id in lines of invoice when sale order is validated and picking is created before invoice ( when in a sale order the picking is created before of to create the invoice, when it's create the invoice, _prepare_order_line_invoice_line method is called and the move_id created is related with the invoice )